packages/css/src/components/accordion/README.md

```diff
@@ -14,6 +14,7 @@ With an accordion, you offer content to users progressively.
 - Don’t hide essential information in an accordion.
 - An accordion consists of several accordion sections.
 - Use a minimum of 3 and a maximum of 10 sections underneath each other.
+- Choose the correct Heading level (`h1` – `h4`) for the accordion sections.
```

Comment:
You choose `1 - 4` so this might be confusing.

Reply:
Yes, good point. Updated.